ivi-sdk-java/src/main/java/games/mythical/ivi/sdk/client/IVIOrderClient.java

```diff
@@ -188,6 +188,28 @@ public IVIFinalizeOrderResponse finalizeUpholdOrder(String orderId,
         return finalizeOrder(orderId, paymentData, fraudSessionId);
     }
 
+    public IVIFinalizeOrderResponse finalizeGr4vyOrder(String orderId,
+                                                       String cardType,
+                                                       String expMonth,
+                                                       String expYear,
+                                                       String instrumentId,
+                                                       String securityCode,
+                                                       String paymentMethodTokenId) throws IVIException {
+
+        var paymentData = PaymentRequestProto.newBuilder()
+                .setGr4Vy(Gr4vyPaymentRequestProto.newBuilder()
+                        .setCardType(cardType)
```

Comment:
I think we should only be sending the `paymentMethodTokenId` through the sdk. The card should already be stored at gr4vy before they call `finalizePayment`.